GLTFSDK/Source/GLTFResourceReader.cpp

```diff
@@ -0,0 +1,60 @@
+// Copyright (c) Microsoft Corporation. All rights reserved.
+// Licensed under the MIT License.
+
+#include <GLTFSDK/GLTFResourceReader.h>
+
+using namespace Microsoft::glTF;
+
+namespace
+{
+	// Conversions of normalized component types to/from floats are explicitly defined in the 2.0 spec
+	inline float ComponentToFloat(const int8_t w)  { return std::max(static_cast<float>(w) / 127.0f, -1.0f); }
+	inline float ComponentToFloat(const uint8_t w) { return static_cast<float>(w) / 255.0f; }
+	inline float ComponentToFloat(const int16_t w) { return std::max(static_cast<float>(w) / 32767.0f, -1.0f); }
+	inline float ComponentToFloat(const uint16_t w){ return static_cast<float>(w) / 65535.0f; }

```

Comment:
I know this code was copy-pasted but can you use std::numeric_limits instead of hardcoding the literal values

Reply:
Isn't hardcoding the literal values correct? These *have* to be exactly as written in the source code to follow the spec.

Reply:
The source code can be whatever we want it to be as long as it is equivalent to the specified 'equations'. Aside from the ridiculous verbosity std::numeric_limits<uint16_t>::max() is at least slightly self-documenting

Reply:
![image](https://user-images.githubusercontent.com/25353211/71374358-4821e580-256f-11ea-8bf1-9d14a38ddb30.png)

I originally wrote it like this to match the spec (attached). I would leave it as-is.